src/app/pages/account/login/page.jsx

```diff
@@ -0,0 +1,118 @@
+'use client'
+import React,{useState} from 'react'
+import styles from './page.module.css'
+import { signIn, useSession } from 'next-auth/react'
+import Link from 'next/link'
+import {AiOutlineUser,AiOutlineLock, AiOutlineEye,AiOutlineEyeInvisible} from 'react-icons/ai'
+import { useRouter } from 'next/navigation'
+import {validateEmail, validatePassword} from '@/utils/formValidations'
+
+const Login = () => {
+  const [passwordVisible, setPasswordVisible] = useState(false)
+  const [validationEmail, setValidationEmail] = useState(false);
+  const [valiEmailMessage, setValiEmailMessage] = useState('');
+  const [emailInput, setEmailInput] = useState('');
+  const [validationPassword, setValidationPassword] = useState(false);
+  const [valiPasswordMessage, setValiPasswordMessage] = useState('');
+  const [passwordInput, setPasswordInput] = useState('');
+  const session = useSession()
+  const router = useRouter()
+  if(session.status === "loading"){
+    return <p>Loading Authentication from Server</p>
+  }
+  if(session.status === "authenticated"){
+    router.push("/");
+}
+  console.log(session)
```

Comment:
Delete this message in console.

Reply:
This was fixed with the latest commit: https://github.com/Les-Jailes/Boutique_Clothing/pull/4/commits/e2b34140a6ca0f3bb80d7e7ac5cb0bb0ed37ffc3